File: 3872-find-most-frequent-vowel-and-consonant/find-most-frequent-vowel-and-consonant.py
class Solution(object):
    def maxFreqSum(self, s):
        """
        :type s: str
        :rtype: int
        """
        vowels="aeiou"
        consonants="bcdfghjklmnpqrstvwxyz"
        v, c = {}, {}
        for i in s:
            if i in vowels:
                v[i]=v.get(i, 0)+1
            elif i in consonants:
                c[i]=c.get(i, 0)+1
        mv=max(v.values()) if v else 0
        mc=max(c.values()) if c else 0
        return mv+mc

File: 3872-find-most-frequent-vowel-and-consonant/find-most-frequent-vowel-and-consonant.cs
public class Solution {
    public int MaxFreqSum(string s) {
        string vowels = "aeiou";
        string consonants = "bcdfghjklmnpqrstvwxyz";
        Dictionary<char, int> v = new Dictionary<char, int>();
        Dictionary<char, int> c = new Dictionary<char, int>();
        foreach(char i in s)
        {
            if(vowels.Contains(i))
            {
                v[i] = v.ContainsKey(i) ? v[i] + 1 : 1;
            }
            else if(consonants.Contains(i))
            {
                c[i] = c.ContainsKey(i) ? c[i] + 1 : 1;
            }
        }
        int mv = v.Count > 0 ? v.Values.Max() : 0;
        int mc = c.Count > 0 ? c.Values.Max() : 0;
        return mv + mc;
    }
}